store-implementation/parquet-store/src/main/java/uk/gov/gchq/gaffer/parquetstore/serialisation/ParquetSerialisationFactory.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2016-2017 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.gov.gchq.gaffer.parquetstore.serialisation;
+
+import uk.gov.gchq.gaffer.serialisation.Serialiser;
+import uk.gov.gchq.gaffer.serialisation.implementation.JavaSerialiser;
+import uk.gov.gchq.gaffer.store.SerialisationFactory;
+
+/**
+ * A <code>ParquetSerialisationFactory</code> holds a list of core serialisers and
+ * is design to provide compatible serialisers for given object classes.
+ */
+public class ParquetSerialisationFactory extends SerialisationFactory {
```

Comment:
The SerialisationFactory has been updating since you wrote this class. There is now a constructor that takes an array of serialisers. Then you can remove the getSerialiser method.